udata_gouvfr/theme/templates/home.html

```diff
@@ -62,6 +62,45 @@ <h3>
             </div>
             {# End last reuses #}
 
+            {# Last inventories #}
+            <div class="col-xs-12">
+                <h3>
+                    <a href="{{ url_for('gouvfr.show_page', slug='donnees-cles-par-sujet') }}"
+                        title="{{ _('Données clés par sujet') }}">
+                        {{ _('Données clés par sujet') }}
+                    </a>
+                </h3>
+                <div class="row">
+                    <ul class="card-list card-list--columned">
+                        <li class="col-sm-4 col-xs-12">
+                            <a class="card-link" href="{{ url_for('gouvfr.show_page', slug='donnees-coronavirus') }}">
+                            <div class="inventory-card green">
+                                <h4 class="inventory-text">Données relatives à la</h4>
```

Comment:
don't use two different header level for the same title (wrong for accessibility). Also, `h2` below is too high, use `h4` (same level as items inside blocks on this page, cf `h3` above also).